### PR TITLE
Add spotter position in live data

### PR DESCRIPTION
**Pull request by avalmas-programize**

### CODE REVIEW

File: packages/api/src/utils/liveData.ts [hunk 1]
```diff
@@ -91,8 +93,11 @@ export const getLiveData = async (reef: Reef): Promise<SofarLiveData> => {
         ),
         wavePeakPeriod: getLatestData(spotterRawData.wavePeakPeriod),
         waveMeanDirection: getLatestData(spotterRawData.waveMeanDirection),
+        longitude: getLatestData(spotterRawData.longitude),
+        latitude: getLatestData(spotterRawData.latitude),
       }
     : {};
+  console.log(spotterData);
```
Comment (ericboucher) [inline packages/api/src/utils/liveData.ts:100]: 👣

File: packages/api/src/workers/dailyData.ts [hunk 2]
```diff
@@ -82,6 +82,8 @@ export async function getDailyData(
           significantWaveHeight: [],
           wavePeakPeriod: [],
           waveMeanDirection: [],
+          latitude: [],
```
Comment (ericboucher) [inline packages/api/src/workers/dailyData.ts:85]: let's not have it in the daily data

File: packages/api/src/utils/liveData.ts [hunk 3]
```diff
@@ -91,8 +93,11 @@ export const getLiveData = async (reef: Reef): Promise<SofarLiveData> => {
         ),
         wavePeakPeriod: getLatestData(spotterRawData.wavePeakPeriod),
         waveMeanDirection: getLatestData(spotterRawData.waveMeanDirection),
+        longitude: getLatestData(spotterRawData.longitude),
```
Comment (ericboucher) [inline packages/api/src/utils/liveData.ts:96]: let's return it as `spotterPosition: {latitude, longitude}` maybe?

File: packages/api/src/utils/liveData.ts (hunk 1 above)
Comment (avalmas-programize) [reply packages/api/src/utils/liveData.ts:100]: nothing to see here

File: packages/api/src/workers/dailyData.ts (hunk 2 above)
Comment (avalmas-programize) [reply packages/api/src/workers/dailyData.ts:85]: ok

File: packages/api/src/utils/liveData.ts (hunk 3 above)
Comment (avalmas-programize) [reply packages/api/src/utils/liveData.ts:96]: ok

**Review by ericboucher — APPROVED** [review]

**Pull request merged**